lib/cchain/provider/testdata/TestXBlock.golden

```diff
@@ -5,12 +5,12 @@
  "Msgs": [
   {
    "SourceChainID": 77,
-   "DestChainID": 1894687307556873581,
```

Comment:
this change is just due to fuzzing that follows a different path and doesn't break things on the wire.